src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -881,15 +881,25 @@ protected override void ProcessRecord()
     /// The core command for setting/changing location and pushing it onto a location stack.
     /// This is the equivalent of the pushd command.
     /// </summary>
-    [Cmdlet(VerbsCommon.Push, "Location", DefaultParameterSetName = "Path", SupportsTransactions = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113370")]
+    [Cmdlet(VerbsCommon.Push, "Location", DefaultParameterSetName = pathSet, SupportsTransactions = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113370")]
     public class PushLocationCommand : CoreCommandBase
     {
         #region Command parameters
 
+        /// <summary>
+        /// The string declaration for the Location parameter set in this command.
+        /// </summary>
+        private const string pathSet = "Path";
```

Comment:
All const names should start with upper case.
```suggestion
        private const string PathParameterSet = "Path";
```